CONTRIBUTING.md

```diff
@@ -1,6 +1,11 @@
 ## Contributing
 
-Chat with us in the [bugbug](https://chat.mozilla.org/#/room/#bugbug:mozilla.org) Matrix room.
-
-1. [Issues marked as `good-first-bug`](https://github.com/mozilla/bugbug/labels/good-first-bug) are self-contained enough that a contributor should be able to work on them.
-2. Issues are considered not assigned, until there is a PR linked to them. Feel free to work on any unassigned issue, you don't need to ask first.
+Steps to contribute a project on Github for beginners:
+1. Create a fork of the project suitable for you on Github.This will create a personal repository on your computer.
+2. Clone the project on your machine and create a branch from master to start working on the project.
+3. Go through the project and make the commits as per the requirements,fix the code/bugs and test your features.
+4. Push the commits so that the project owner could implement the features corrected by you.
+5. Make a pull request on Github.
+6. Discuss the project with the owner, if any further changes are needed and commit those changes till the code is rectified.
+7. The project owner would approve and merge the pull requests to the repository.
+8. Sync the updated master back to your fork.
```

Comment:
After the changes have been merged, to the bugbug main repository, you can delete the feature_branch on your forked repository. Head over to the main/master branch on your machine and then pull the latest changes. (Step 6) [Reference Step 6.]

Sync the updated master back to your fork. (Include the command to push the pulled changes to the fork.)

You are free to pick up any other issue and/or making further contributions.

Reply:
### Join Our Chat
Chat with us in the [bugbug](https://chat.mozilla.org/#/room/#bugbug:mozilla.org) Matrix room.

Reply:
Thanks @srishtig2412 , great guidelines.